src/main/resources/electronic-data-mapping.json

```diff
@@ -0,0 +1,291 @@
+{
+  "orderMappings":
+  [
+    {
+      "orderType": "ListedElectronicMonograph",
+      "mappings":
+      [
+        {
+          "field":"ACCOUNT_NUMBER",
+          "dataSource":{
+            "from":"//SubAccount",
+            "default":"0"
+          }
+        },
+        {
+          "field":"ACQUISITION_METHOD",
+          "dataSource":{
+            "default":"PVA"
+          }
+        },
+        {
+          "field":"APPROVAL_STATUS",
+          "dataSource":{
+            "default":"A",
+            "translation": "lookupWorkflowStatusId",
+            "translateDefault": true
+          }
+        },
+        {
+          "field":"CLAIM",
+          "dataSource": {
+            "default": "true"
+          }
+        },
+        {
+          "field":"COLLECTION",
+          "dataSource": {
+            "default": "false"
+          }
+        },
+        {
+          "field":"CONTRIBUTOR",
+          "dataSource": {
+            "from": "//datafield[@tag='100']/*",
+            "combinator":"concat"
+          }
+        },
+        {
+          "field":"CREATED_DATE",
+          "dataSource": {
+            "from": "//OrderPlaced",
+            "translation": "toDate"
+          }
+        },
+        {
+          "field":"CURRENCY",
+          "dataSource":{
+            "from":"//ListPrice/Currency",
+            "default":"USD"
+          }
+        },
+        {
+          "field":"FUND_CODE",
+          "dataSource": {
+            "from": "//FundCode",
+            "translation": "lookupFundId"
+          }
+        },
+        {
+          "field":"LIST_PRICE",
+          "dataSource":{
+            "from":"//ListPrice/Amount",
+            "default":"0",
+            "translation": "toDouble"
+          }
+        },
+        {
+          "field":"LOCATION",
+          "dataSource": {
+            "from": "//Location"
+          }
+        },
+        {
+          "field":"MATERIAL_TYPE",
+          "dataSource": {
+            "from": "//LocalData[Description='LocalData1']/Value",
+            "translation": "lookupMaterialTypeId"
```

Comment:
Is this lookup needed? I'm not aware of this requirement, although we already had the function "lookupMaterialTypeId" written up previously.

Reply:
The assumption here is that `LocalData1` contains something like "book" and that would need to be looked up to get the material type ID to store in the details schema. The problem here is that "material_type" in the details schema is just a string. What string should go in there is unclear. I would expect that it would be the ID of the material type. If it is not the case, we can easily remove the translation.

Reply:
That was my understand as well Matt.